modules/study/src/main/StudyForm.scala

```diff
@@ -88,9 +88,7 @@ object StudyForm {
             name = Chapter.Name((index == 0) ?? name),
             variant = variantStr,
             pgn = onePgn.some,
-            orientation =
-              if (pgns.sizeIs > 1) "auto"
-              else (orientationStr.flatMap(chess.Color.fromName) | chess.White).name,
+            orientation = orientationStr | "white",
```

Comment:
Validation seems a bit unnecessary here since it will be done again in `ChapterMaker`  but if you want to be safe, `orientationStr.flatMap(chess.Color.fromName).fold("automatic")(_.name)` would probably be the easiest option.